pkg/resource/obtenantrestore_task.go

```diff
@@ -36,7 +38,14 @@ import (
 
 // OBTenantManager tasks completion
 
-func (m *OBTenantManager) CreateTenantRestoreJob() error {
+func (m *OBTenantManager) generateRestoreOption() string {
+	poolList := m.generateSpecPoolList(m.OBTenant.Spec.Pools)
+	primaryZone := m.generateSpecPrimaryZone(m.OBTenant.Spec.Pools)
+	locality := m.generateLocality(m.OBTenant.Spec.Pools)
+	return fmt.Sprintf("pool_list=%s&primary_zone=%s&locality=%s", strings.Join(poolList, ","), primaryZone, locality)
+}
+
+func (m *OBTenantManager) CreateTenantRestoreJobCR() error {
```

Comment:
use CreateTenantRestoreJob is ok

Reply:
I thought `CreateTenantRestoreJob` is semantically ambiguous that I cannot figure out it would create a Job in OB or a Job in K8s